cmd/restic/cmd_find.go

```diff
@@ -637,7 +642,11 @@ func runFind(ctx context.Context, opts FindOptions, gopts GlobalOptions, args []
 	}
 
 	sort.Slice(filteredSnapshots, func(i, j int) bool {
-		return filteredSnapshots[i].Time.Before(filteredSnapshots[j].Time)
+		if opts.Reverse {
```

Comment:
Our linter rules expect this to be written as follows:
```go
		if opts.Reverse {
			return filteredSnapshots[i].Time.Before(filteredSnapshots[j].Time)
		}
		return filteredSnapshots[i].Time.After(filteredSnapshots[j].Time)
```

You can manually call `golangci-lint run ./...` to get the linting results (or setup your editor accordingly)